tests/ui/traits/const-traits/cross-crate.stock.stderr

```diff
@@ -1,9 +1,10 @@
 error[E0658]: cannot call conditionally-const method `<cross_crate::Const as cross_crate::MyTrait>::func` in constant functions
-  --> $DIR/cross-crate.rs:22:5
+  --> $DIR/cross-crate.rs:22:11
    |
 LL |     Const.func();
-   |     ^^^^^^^^^^^^
+   |           ^^^^^^
    |
+   = note: calls in constant functions are limited to constant functions, tuple structs and tuple variants
    = note: see issue #67792 <https://github.com/rust-lang/rust/issues/67792> for more information
    = help: add `#![feature(const_trait_impl)]` to the crate attributes to enable
```

Comment:
Is this the error that shows up when the function is conditionally-const and we don't have enough trait bounds to know whether it is actually const, or when it is conditionally-const and we do have the trait bounds but we need to enable a nightly feature to use it?

Reply:
The latter, but we don't distinguish the two per my comment above. I'll probably try to make this less misleading, but I'd prefer to do it as a follow-up. I think we can keep iterating on these errors as need (and as users complain).